form.html

```diff
@@ -0,0 +1,159 @@
+                    <label class="form__label-radio" for="answer-positive">Rather positive</label>
+                  </div>
+                </li>
+                <li class="form__radio-item">
+                  <div class="form__negative-answer">
+                    <input class="form__input-radio" id="answer-negative" name="answer" type="radio" value="rather-negative"/>
+                    <label class="form__label-radio" for="answer-negative">Rather negative</label>
+                  </div>
+                </li>
+                <li class="form__radio-item">
+                  <div class="form__difficult-answer">
+                    <input class="form__input-radio" id="answer-difficult" name="answer" type="radio" value="difficult-answer"/>
+                    <label class="form__label-radio" for="answer-difficult">Difficult to answer</label>
+                  </div>
+                </li>
+              </ul>
+            </fieldset>
+          </section>
+
+          <section class="form__section">
+            <h2 class="form__header">Sights visited:</h2>
+            <ul class="form__checkbox-list">
+              <li class="form__checkbox-item">
+                <div class="form__visited-places">
+                  <input class="form__input-checkbox" id="bridge" name="bridge" type="checkbox" value="devils-bridge">
+                  <label class="form__label-checkbox" for="bridge">The Devil's Bridge</label>
+                </div>
+              </li>
+              <li class="form__checkbox-item">
+                <div class="form__visited-places">
+                  <input class="form__input-checkbox" id="park" name="park" type="checkbox" value="slide-park">
+                  <label class="form__label-checkbox" for="park">Slide-park</label>
+                </div>
+              </li>
+              <li class="form__checkbox-item">
+                <div class="form__visited-places">
+                  <input class="form__input-checkbox" id="mountain" name="mountain" type="checkbox" value="mountain-bell">
+                  <label class="form__label-checkbox" for="mountain">Mountain-bell</label>
+                </div>
+              </li>
+              <li class="form__checkbox-item">
+                <div class="form__visited-places">
+                  <input class="form__input-checkbox" id="cliffs" name="cliffs" type="checkbox" value="red-cliffs">
+                  <label class="form__label-checkbox" for="cliffs">Red cliffs</label>
+                </div>
+              </li>
+            </ul>
+          </section>
+
+          <section class="form__section">
+            <div class="form__feedback">
+              <label class="form__label-textarea" for="feedback">Describe your emotions:</label>
+              <textarea class="form__input-textarea" id="feedback" name="feedback"></textarea>
+              <p class="form__paragraph--shaddow">* - required fields</p>
```

Comment:
cheeky modifier class without block class, should be an element. Why should it be part of the form__label-textarea.  Also looking that we have that, we need to add * to some of the fields, I don't see on the preview of the design what's required. If it's not in PSD sours either, then add required red colour asterisk after for name, surname and email. Make use of HTML required attribute as well.

Reply:
it appears in the tablet and desktop psd 
![image](https://user-images.githubusercontent.com/6135781/32416753-55092f8a-c246-11e7-802e-f7214d4b9b1f.png)

As previously if i want to use form__paragraph--shadow it should fallow form__paragraph first yes?

Reply:
I think it's more of form block component, you can stick it at the end of form, or put it in the same block with button. It's on the same level with button. makes more sense